data/transactions/logic/assembler_test.go

```diff
@@ -793,7 +812,7 @@ func TestOpUint(t *testing.T) {
 	for v := uint64(1); v <= AssemblerMaxVersion; v++ {
 		t.Run(fmt.Sprintf("v=%d", v), func(t *testing.T) {
 			ops := newOpStream(v)
-			ops.IntLiteral(0xcafebabe)
+			ops.intLiteral(0xcafebabe)
```

Comment:
maybe 8badf00d is less cringe

Reply:
fair enough. I surely didn't write that, but now that I've changed the line, I'll be git blamed!